docs/content/preview/architecture/docdb-sharding/tablet-splitting.md

```diff
@@ -215,13 +215,20 @@ For details on the architecture design, see [Automatic resharding of data with t
 
 #### Enable automatic tablet splitting
 
-To enable automatic tablet splitting, use the `yb-master` [`--enable_automatic_tablet_splitting`](../../../reference/configuration/yb-master/#enable-automatic-tablet-splitting) flag and specify the associated flags to configure when tablets should split, and use `yb-tserver` [`--enable_automatic_tablet_splitting`](../../../reference/configuration/yb-tserver/#enable-automatic-tablet-splitting). The flag usage must match on all `yb-master` and `yb-tserver` configurations of a YugabyteDB cluster.
+When automatic tablet splitting is enabled, newly-created tables with [hash sharding](../../../architecture/docdb-sharding/sharding/#hash-sharding) have one tablet per node by default.
 
-When automatic tablet splitting is enabled, newly-created tables with [hash sharding](../../../architecture/docdb-sharding/sharding/#hash-sharding) have one shard **per _yb-tserver_** by default; from version `2.14.10` such tables have one shard (for servers with up to 2 CPU cores) or two shards (for servers with up to 4 CPU cores) **per _cluster_** by default.
+The following additional defaults apply:
+
+* From version 2.14.10, for servers with up to 2 CPU cores, newly-created tables have one tablet or, for servers with up to 4 CPU cores, two tablets **per _cluster_** by default.
+* From version 2.18, for servers with up to 2 CPU cores, newly-created tables have one tablet per table, and servers with up to 4 CPU cores have 2 tablets per table.
```

Comment:
These lines describe the same behaviour, you should keep only one line. The behaviour was changed starting form 2.14.10.

Reply:
Oh, so this started with 2.14.10, not 2.18, it is only the setting of the default that changed with 2.18. OK that clarifies things

Reply:
It was initially implemented for 2.19 (master) and then backported to 2.18, 2.16 and 2.14. Probably it is OK to have 2.14.10. But originally each docs version mentioned the corresponding backport version: for 2.14 it was 2.14.10, for 2.16 - 2.16.5 and so on.